casr/src/bin/casr-afl.rs

```diff
@@ -85,6 +136,14 @@ fn main() -> Result<()> {
                 .long("no-cluster")
                 .help("Do not cluster CASR reports")
         )
+        .arg(
+            Arg::new("ARGS")
+                .action(ArgAction::Set)
+                .required(false)
+                .num_args(1..)
+                .last(true)
+                .help("Add \"-- ./gdb_fuzz_target <arguments>\" for additional triaging with casr-gdb"),
```

Comment:
We need more clear message here. I suppose we just add gdb reports for already triaged asan crash reports. Also fix README.md and usage.md